torch/_dynamo/utils.py

```diff
@@ -1584,3 +1584,17 @@ def convert(obj):
     if isinstance(___tmp_0, tuple):
         return tuple([convert(obj) for obj in ___tmp_0])
     return convert(___tmp_0)
+
+
+def to_torch_np_ndarray(___graph_out_0):
```

Comment:
It's pretty strange to use this argument naming, since you never actually call this from generated bytecode